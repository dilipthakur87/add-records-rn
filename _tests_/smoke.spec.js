describe('truth', () => {
    it('is ture', () => {
        expect(true).toEqual(true);
    });
})