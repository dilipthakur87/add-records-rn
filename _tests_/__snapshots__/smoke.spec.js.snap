// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Modal
  animationType="slide"
  hardwareAccelerated={false}
  transparent={true}
  visible={true}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
        "marginTop": 22,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "#fff",
          "borderRadius": 7,
          "elevation": 5,
          "height": 1156.1757719714965,
          "margin": 20,
          "shadowColor": "#000",
          "shadowOffset": Object {
            "height": 2,
            "width": 0,
          },
          "shadowOpacity": 0.25,
          "shadowRadius": 3.84,
          "width": 691.2114014251781,
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "row",
            "justifyContent": "center",
            "marginTop": 40.973871733966746,
            "width": "100%",
          }
        }
      >
        <Text
          style={
            Object {
              "alignSelf": "center",
              "color": "#1469AF",
              "fontSize": 35.62945368171021,
            }
          }
        >
          User Input Modal
        </Text>
        <View
          accessible={true}
          focusable={true}
          isTVSelectable={true}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "opacity": 1,
              "position": "absolute",
              "right": 21,
            }
          }
        >
          <Image
            source={1}
            style={
              Object {
                "height": 35.62945368171021,
                "resizeMode": "contain",
                "tintColor": "#000",
                "width": 35.62945368171021,
              }
            }
          />
        </View>
      </View>
      <View
        style={
          Object {
            "flex": 1,
            "flexDirection": "column",
            "justifyContent": "center",
          }
        }
      >
        <View
          style={
            Object {
              "margin": 39.23529411764706,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#536278",
                "fontSize": 18,
                "paddingBottom": 5,
                "paddingRight": 10,
              }
            }
          >
            Name
            :
          </Text>
          <TextInput
            allowFontScaling={true}
            editable={true}
            onChangeText={[Function]}
            rejectResponderTermination={true}
            style={
              Object {
                "backgroundColor": "white",
                "borderColor": "#9ea7be",
                "borderRadius": 5,
                "borderWidth": 1,
                "color": "#1b3964",
                "fontSize": 18,
                "padding": 5,
              }
            }
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Object {
              "margin": 39.23529411764706,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#536278",
                "fontSize": 18,
                "paddingBottom": 5,
                "paddingRight": 10,
              }
            }
          >
            Country
            :
          </Text>
          <TextInput
            allowFontScaling={true}
            editable={true}
            onChangeText={[Function]}
            rejectResponderTermination={true}
            style={
              Object {
                "backgroundColor": "white",
                "borderColor": "#9ea7be",
                "borderRadius": 5,
                "borderWidth": 1,
                "color": "#1b3964",
                "fontSize": 18,
                "padding": 5,
              }
            }
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Object {
              "margin": 39.23529411764706,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#536278",
                "fontSize": 18,
                "paddingBottom": 5,
                "paddingRight": 10,
              }
            }
          >
            Phone Number
            :
          </Text>
          <TextInput
            allowFontScaling={true}
            editable={true}
            onChangeText={[Function]}
            rejectResponderTermination={true}
            style={
              Object {
                "backgroundColor": "white",
                "borderColor": "#9ea7be",
                "borderRadius": 5,
                "borderWidth": 1,
                "color": "#1b3964",
                "fontSize": 18,
                "padding": 5,
              }
            }
            underlineColorAndroid="transparent"
            value=""
          />
        </View>
        <View
          style={
            Object {
              "borderColor": "#9ea7be",
              "borderRadius": 5,
              "borderWidth": 1,
              "fontSize": 18,
              "marginHorizontal": 45,
              "marginTop": 40,
              "padding": 5,
            }
          }
        >
          <View
            style={
              Array [
                Object {
                  "alignSelf": "stretch",
                },
                undefined,
              ]
            }
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "opacity": 1,
                }
              }
              testID="ios_touchable_wrapper"
            >
              <View
                pointerEvents="box-only"
              >
                <TextInput
                  allowFontScaling={true}
                  editable={false}
                  rejectResponderTermination={true}
                  style={
                    Array [
                      undefined,
                      Object {
                        "color": "#c7c7cd",
                      },
                    ]
                  }
                  testID="text_input"
                  underlineColorAndroid="transparent"
                  value="Please select your favourite phone brand."
                />
              </View>
            </View>
            <Modal
              hardwareAccelerated={false}
              onOrientationChange={[Function]}
              supportedOrientations={
                Array [
                  "portrait",
                  "landscape",
                ]
              }
              testID="ios_modal"
              transparent={true}
              visible={false}
            >
              <View
                accessible={true}
                focusable={true}
                isTVSelectable={true}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "flex": 1,
                    "opacity": 1,
                  }
                }
                testID="ios_modal_top"
              />
              <View
                style={
                  Array [
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "#f8f8f8",
                      "borderTopColor": "#dedede",
                      "borderTopWidth": 1,
                      "flexDirection": "row",
                      "height": 45,
                      "justifyContent": "space-between",
                      "paddingHorizontal": 10,
                      "zIndex": 2,
                    },
                    undefined,
                  ]
                }
                testID="input_accessory_view"
              >
                <View
                  style={
                    Array [
                      Object {
                        "flexDirection": "row",
                      },
                      undefined,
                    ]
                  }
                >
                  <View
                    accessible={true}
                    focusable={true}
                    isTVSelectable={true}
                    onClick={[Function]}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "opacity": 1,
                      }
                    }
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "backgroundColor": "transparent",
                            "borderColor": "#a1a1a1",
                            "borderRightWidth": 1.5,
                            "borderTopWidth": 1.5,
                            "height": 15,
                            "width": 15,
                          },
                          undefined,
                          Object {
                            "marginLeft": 11,
                            "transform": Array [
                              Object {
                                "translateY": 4,
                              },
                              Object {
                                "rotate": "-45deg",
                              },
                            ],
                          },
                          undefined,
                          Object {},
                        ]
                      }
                    />
                  </View>
                  <View
                    accessible={true}
                    focusable={true}
                    isTVSelectable={true}
                    onClick={[Function]}
                    onResponderGrant={[Function]}
                    onResponderMove={[Function]}
                    onResponderRelease={[Function]}
                    onResponderTerminate={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    style={
                      Object {
                        "opacity": 1,
                      }
                    }
                  >
                    <View
                      style={
                        Array [
                          Object {
                            "backgroundColor": "transparent",
                            "borderColor": "#a1a1a1",
                            "borderRightWidth": 1.5,
                            "borderTopWidth": 1.5,
                            "height": 15,
                            "width": 15,
                          },
                          undefined,
                          Object {
                            "marginLeft": 22,
                            "transform": Array [
                              Object {
                                "translateY": -5,
                              },
                              Object {
                                "rotate": "135deg",
                              },
                            ],
                          },
                          undefined,
                          Object {},
                        ]
                      }
                    />
                  </View>
                </View>
                <View
                  accessible={true}
                  focusable={true}
                  hitSlop={
                    Object {
                      "bottom": 4,
                      "left": 4,
                      "right": 4,
                      "top": 4,
                    }
                  }
                  isTVSelectable={true}
                  onClick={[Function]}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "opacity": 1,
                    }
                  }
                  testID="done_button"
                >
                  <View
                    testID="needed_for_touchable"
                  >
                    <Text
                      allowFontScaling={false}
                      style={
                        Array [
                          Object {
                            "color": "#007aff",
                            "fontSize": 17,
                            "fontWeight": "600",
                            "paddingRight": 11,
                            "paddingTop": 1,
                          },
                          undefined,
                          Object {},
                        ]
                      }
                      testID="done_text"
                    >
                      Done
                    </Text>
                  </View>
                </View>
              </View>
              <View
                style={
                  Array [
                    Object {
                      "backgroundColor": "#d0d4da",
                      "justifyContent": "center",
                    },
                    Object {
                      "height": 215,
                    },
                    undefined,
                  ]
                }
              >
                <View>
                  <RCTPicker
                    items={
                      Array [
                        Object {
                          "label": "Please select your favourite phone brand.",
                          "textColor": undefined,
                          "value": "",
                        },
                        Object {
                          "label": "iPhone",
                          "textColor": undefined,
                          "value": "iPhone",
                        },
                        Object {
                          "label": "Samsung",
                          "textColor": undefined,
                          "value": "Samsung",
                        },
                        Object {
                          "label": "Nokia",
                          "textColor": undefined,
                          "value": "Nokia",
                        },
                        Object {
                          "label": "Huawei",
                          "textColor": undefined,
                          "value": "Huawei",
                        },
                        Object {
                          "label": "Oppo",
                          "textColor": undefined,
                          "value": "Oppo",
                        },
                        Object {
                          "label": "Xiaomi",
                          "textColor": undefined,
                          "value": "Xiaomi",
                        },
                        Object {
                          "label": "Lenovo",
                          "textColor": undefined,
                          "value": "Lenovo",
                        },
                        Object {
                          "label": "Vivo",
                          "textColor": undefined,
                          "value": "Vivo",
                        },
                        Object {
                          "label": "Others",
                          "textColor": undefined,
                          "value": "Others",
                        },
                      ]
                    }
                    onChange={[Function]}
                    onResponderTerminationRequest={[Function]}
                    onStartShouldSetResponder={[Function]}
                    selectedIndex={0}
                    style={
                      Array [
                        Object {
                          "height": 216,
                        },
                        undefined,
                      ]
                    }
                    testID="ios_picker"
                  />
                </View>
              </View>
            </Modal>
          </View>
        </View>
        <View
          style={
            Object {
              "alignItems": "center",
              "margin": 39.23529411764706,
            }
          }
        >
          <View
            accessible={true}
            focusable={true}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#3e98e0",
                "borderRadius": 5,
                "height": 45,
                "justifyContent": "center",
                "marginBottom": 78.47058823529412,
                "marginTop": 39.23529411764706,
                "opacity": 1,
                "width": 200,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "white",
                  "fontSize": 18,
                  "fontWeight": "500",
                }
              }
            >
              Submit
            </Text>
          </View>
        </View>
      </View>
    </View>
  </View>
</Modal>
`;
